packages/repository-tests/package.json

```diff
@@ -23,15 +23,18 @@
     "@loopback/build": "^1.7.1",
     "@loopback/repository": "^1.12.0",
     "@types/debug": "^4.1.5",
-    "@types/node": "^10.14.17"
+    "@types/node": "^10.14.17",
+    "@types/lodash": "^4.14.137"
   },
   "dependencies": {
+    "@loopback/context": "^1.21.3",
     "@loopback/testlab": "^1.7.4",
-    "@types/debug": "^4.1.5",
+    "loopback-datasource-juggler": "^4.12.0",
```

Comment:
AFAICT, `loopback-datasource-jugler` is not used directly anywhere in the new code, can we remove it from the dependencies please?